### PR TITLE
Make satellite apps non-optional

### PR DESCRIPTION
**Pull request by chosak**
This change moves our "optional" satellite apps into the main list of Python libraries, treating them the same as our other packages.

This is now possible as all satellites are compatible with both Python 2 and 3.

## Testing

This will add the satellite apps to the list of packages installed by tox, and so this is a change that expands what Travis tests (hopefully in a good way).

## Todos

I've placed the "hold" label on this PR because this is a substantial change to how things work now locally, and we should make sure that the documentation is updated appropriately about this. Currently satellite app information is split across three pages (["Running in Docker"](https://cfpb.github.io/cfgov-refresh/running-docker/#work-on-satellite-apps), ["Satellite Repos"](https://cfpb.github.io/cfgov-refresh/satellite-repos/), and ["Development Tips"](https://cfpb.github.io/cfgov-refresh/development-tips/#tip-loading-satellite-apps)). These should be consolidated as best as possible.

With this change, developers using a local virtualenv will need to know how to `pip install` a local checkout in order to work on it, and then how to switch back to the master version. Similarly, Docker users will need to understand how the "develop-apps" directory works as a second installation that overrides the version that gets installed in the standard environment.

## Checklist

- [X] PR has an informative and human-readable title
- [X] Changes are limited to a single goal (no scope creep)
- [X] Code can be automatically merged (no conflicts)
- [X] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [X] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [X] Project documentation has been updated
- [X] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

**Comment by chosak** [comment]
I thought of a potential problem with doing this, which is [this line](https://github.com/cfpb/teachers-digital-platform/blob/master/install_cfgov_refresh.sh#L6) in the TDP setup script. I would hate to have to keep a separate requirements file just to support that. @willbarton can you think of any way around this?

**Comment by willbarton** [comment]
🙈

**Comment by willbarton** [comment]
@chosak How about instead of adding everything to libraries.txt, we add a satellites.txt, and that lets us be a little more fine-grained about how we use it. 

At least until we get TDP moved into cfgov-refresh.

**Comment by chosak** [comment]
I opened [a PR to TDP](https://github.com/cfpb/teachers-digital-platform/pull/350) that tries to address this a different way, by having TDP filter itself out when installing. This keeps the TDP-specific logic to TDP, instead of requiring us to maintain a separate file solely for TDP's sake.

**Comment by chosak** [comment]
> I've placed the "hold" label on this PR because this is a substantial change to how things work now locally, and we should make sure that the documentation is updated appropriately about this. Currently satellite app information is split across three pages ("Running in Docker", "Satellite Repos", and "Development Tips"). These should be consolidated as best as possible.
> 
> With this change, developers using a local virtualenv will need to know how to pip install a local checkout in order to work on it, and then how to switch back to the master version. Similarly, Docker users will need to understand how the "develop-apps" directory works as a second installation that overrides the version that gets installed in the standard environment.

I'm removing "hold" from this PR as I've added some documentation that hopefully covers the changes as part of a new "Related Projects" page that consolidates mention of satellite apps. Further reviews and suggestions are welcome.

image

File: docs/related-projects.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+For this reason, the Docker setup provides the ability to use the local
+[`cfgov-refresh/develop-apps`](https://github.com/cfpb/cfgov-refresh/tree/master/develop-apps)
+subdirectory as place to put local copies of Python packages.
+
+Any packages put there (e.g. via a `git clone` of a satellite apps' repo)
+will be automatically added to the
+[PYTHONPATH](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH) 
+in the Python containers.
+Apps ntained within will be importable from Python running in the Docker
```
Comment (schbetsy) [inline docs/related-projects.md]: > Apps ntained within

Minor typo here

Comment (chosak) [reply docs/related-projects.md]: Thanks @schbetsy, fixed in c1d9a99.

**Pull request merged**